### PR TITLE
Maintenance

### LINKED ISSUE #10: npm RunKit example broken?
https://npm.runkit.com/ml-levenberg-marquardt

This seems to spew errors:

screen shot 2018-02-02 at 16 45 10

### PR DESCRIPTION
**Pull request by maasencioh**
This also should fix #10

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -1 +1,5 @@
 extends: 'eslint-config-cheminfo'
+parserOptions:
+  sourceType: module
+env:
+  jest: true
```
Comment (targos) [inline .eslintrc.yml]: you can remove the env parameter. It is now correctly configured in the cheminfo config

File: package.json [hunk 2]
```diff
@@ -26,24 +27,30 @@
...
   "runkitExampleFilename": "./runkit.js",
   "homepage": "https://github.com/mljs/levenberg-marquardt#readme",
   "devDependencies": {
-    "codecov": "^2.2.0",
-    "eslint": "^4.2.0",
-    "eslint-config-cheminfo": "^1.6.0",
+    "babel-jest": "^22.1.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
```
Comment (targos) [inline package.json]: I think you don't need babel-jest, babel-loader and babel-preset-env

File: package.json [hunk 3]
```diff
@@ -26,24 +27,30 @@
...
+    "codecov": "^2.3.1",
+    "eslint": "^4.17.0",
+    "eslint-config-cheminfo": "^1.15.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jest": "^21.7.0",
     "eslint-plugin-no-only-tests": "^2.0.0",
```
Comment (targos) [inline package.json]: you can remove this dependency

File: rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export default {
+  input: 'src/index.js',
+  output: {
+    format: 'cjs',
+    file: 'lib/index.js'
+  },
+  exports: 'named'
```
Comment (targos) [inline rollup.config.js]: What about default export instead? There is no named export in this library

File: runkit.js [hunk 5]
```diff
@@ -1,22 +1,22 @@
-const levenbergMarquardt = require('ml-levenberg-marquardt');
+const levenbergMarquardt = require('ml-levenberg-marquardt').default;
```
Comment (targos) [inline runkit.js]: If you change the rollup config to default exports, you can remove this `.default`

Comment (maasencioh) [reply runkit.js]: this is a great tip, thanks!

**Comment by maasencioh** [comment]
merci @targos

**Pull request merged**